Tools/Scripts/libraries/reporelaypy/reporelaypy/checkout.py

```diff
@@ -219,10 +226,26 @@ def push_update(self, branch=None, tag=None, remote=None, track=False):
             return False
 
         return not run(
-            [self.repository.executable(), 'push', remote, branch, '-f'],
+            [self.repository.executable(), 'push', remote, '{}:{}'.format(branch, dest_branch or branch)],
             cwd=self.repository.root_path,
         ).returncode
 
+    def was_updated(self, branch=None, tag=None, remote=None, track=False):
```

Comment:
This function name sounds like it will return a boolean, but it's a void function. Maybe `forward_update` or if you have more accurate function name?

Reply:
Will change, that's better terminology